src/palace/manager/integration/catalog/marc/annotator.py

```diff
@@ -69,8 +69,29 @@ def marc_record(
         cls.add_summary(record, work)
         cls.add_genres(record, work)
 
+        # Normalize record should always be called after all additions to the
+        # record have been completed.
+        cls._normalize_record(record)
         return record
 
+    @classmethod
+    def _normalize_record(cls, record: Record) -> None:
+        """Make any necessary adjustments to the record.
+
+        This method is the place to add record updates that depend on
+        information from the MARC record itself.
+        """
+
+        # Set the `245` first indicator to "1", if a Main Entry (`1xx`) is present.
```

Comment:
Might be worth adding https://www.loc.gov/marc/bibliographic/bd245.html to the comment, to help people like me that have no idea what the 245 field does.